### PR TITLE
Hiring ruby programmers

### PR DESCRIPTION
**Pull request by kucho**

### CODE REVIEW

File: week-1/day-1/hiring_programmers.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class Candidate
+  def initialize(years_of_experience, github_points, languages_worked_with, applied_recently, age)
+    @years_of_experience = years_of_experience
+    @github_points = github_points
+    @languages_worked_with = languages_worked_with
+    @applied_recently = applied_recently
+    @age = age
```
Comment (seussz) [inline week-1/day-1/hiring_programmers.rb:7]: Wow! O.O Your code looks great! Can you explain me why you use @ at the begin of each variable?

File: week-1/day-1/hiring_programmers.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  end
+
+  attr_accessor :years_of_experience, :github_points, :languages_worked_with, :applied_recently, :age
```
Comment (seussz) [inline week-1/day-1/hiring_programmers.rb:10]: Please, can you explain me why symbols are your option?

File: week-1/day-1/hiring_programmers.rb (hunk 1 above)
Comment (kucho) [reply week-1/day-1/hiring_programmers.rb:7]: It's called an instance variable. When we create a new instance of a class we can pass information through the functions arguments like this:

`ada_object = Person.new("Ada")`

Now, in order to make the object preserve that information we need to assign the local variable we are receiving ("Ada") to the instance variable name with a  `@something`.

You can read more about it here: http://ruby-for-beginners.rubymonstas.org/writing_classes/initializers.html

File: week-1/day-1/hiring_programmers.rb (hunk 2 above)
Comment (kucho) [reply week-1/day-1/hiring_programmers.rb:10]: Sorry, I don't understand your question. Can you rephrase it?

Comment (seussz) [reply week-1/day-1/hiring_programmers.rb:10]: My bad. I mean why you prefer use `:variable` instead of `variable` (without colon)?

File: week-1/day-1/hiring_programmers.rb (hunk 1 above)
Comment (seussz) [reply week-1/day-1/hiring_programmers.rb:7]: Thank you!

File: week-1/day-1/hiring_programmers.rb (hunk 2 above)
Comment (kucho) [reply week-1/day-1/hiring_programmers.rb:10]: The `attr_accessor` method creates a setter and a getter for the variables and it uses symbols as parameters although converts strings to symbols too.

If you wanna know what's happening behind the scenes you should read this interesting article: https://medium.com/@amliving/diving-into-rubys-attr-accessor-d34e2ccc5477
